notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb

```diff
@@ -438,7 +457,7 @@
       "source": [
         "#### Quick peek at your data\n",
         "\n",
-        "This tutorial uses a version of the Salads dataset that is stored in a public Cloud Storage bucket, using a CSV index file.\n",
+        "This tutorial uses a version of salads dataset which is copied to the project's Cloud Storage Bucket.\n",
```

Comment:
- change: "Cloud Storage Bucket"
- to: "Cloud Storage bucket" (i.e., change "Bucket" to lower case)